shotover-proxy/tests/helpers/cassandra.rs

```diff
@@ -274,42 +568,98 @@ impl ResultValue {
                 ValueType::COUNTER => ResultValue::Counter(value.get_i64().unwrap()),
                 ValueType::VARINT => ResultValue::VarInt(value.get_bytes().unwrap().to_vec()),
                 ValueType::TINY_INT => ResultValue::TinyInt(value.get_i8().unwrap()),
-                ValueType::SET => {
-                    ResultValue::Set(value.get_set().unwrap().map(ResultValue::new).collect())
-                }
+                ValueType::SET => ResultValue::Set(
+                    value
+                        .get_set()
+                        .unwrap()
+                        .map(ResultValue::new_from_cpp)
+                        .collect(),
+                ),
                 // despite the name get_set is used by SET, LIST and TUPLE
-                ValueType::LIST => {
-                    ResultValue::List(value.get_set().unwrap().map(ResultValue::new).collect())
-                }
-                ValueType::TUPLE => {
-                    ResultValue::Tuple(value.get_set().unwrap().map(ResultValue::new).collect())
-                }
+                ValueType::LIST => ResultValue::List(
+                    value
+                        .get_set()
+                        .unwrap()
+                        .map(ResultValue::new_from_cpp)
+                        .collect(),
+                ),
+                ValueType::TUPLE => ResultValue::Tuple(
+                    value
+                        .get_set()
+                        .unwrap()
+                        .map(ResultValue::new_from_cpp)
+                        .collect(),
+                ),
                 ValueType::MAP => ResultValue::Map(
                     value
                         .get_map()
                         .unwrap()
-                        .map(|(k, v)| (ResultValue::new(k), ResultValue::new(v)))
+                        .map(|(k, v)| (ResultValue::new_from_cpp(k), ResultValue::new_from_cpp(v)))
                         .collect(),
                 ),
                 ValueType::UNKNOWN => todo!(),
                 ValueType::CUSTOM => todo!(),
                 ValueType::UDT => todo!(),
                 ValueType::TUPLE => todo!(),
+                ValueType::TEXT => unimplemented!(),
```

Comment:
Can we include in the unimplemented!() a message explaining that text is represented by the same id as varchar at the protocol level and therefore will never be instantiated by the datastax cpp driver